docs/core/plugins/sieve.md

```diff
@@ -383,153 +465,114 @@ when a plugin called `sieve_storage_ldap` is loaded.
 
 #### Configuration
 
-If support for the `ldap` location type is compiled as a plugin, it
-needs to be added to the sieve_plugins setting before it can be used,
-e.g.:
-
-```
-sieve_plugins = sieve_storage_ldap
-```
-
-The `ldap` script location syntax is specified as follows:
+The `ldap` storage driver supports all settings described in
+[Common Settings](#common-settings). The following settings apply to this script
+storage driver:
 
-```
-location = ldap:<config-file>[;<option>[=<value>][;...]]
-```
+<SettingsComponent tag="sieve-storage-ldap" />
 
```

Comment:
Maybe a second SettingsComponent with tag="ldap"

Reply:
No, some Dovecot ldap settings are not currently recognized by Sieve ldap driver. I am documenting as-is, not what it should be.